x-pack/solutions/observability/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts

```diff
@@ -213,6 +211,10 @@ export function registerRoutes({
           opts.body.attributes.data = error?.data;
         }
 
+        if (opts.statusCode >= 500) {
```

Comment:
Nit / Q: Is in this case `opts.statusCode` always `500` based on line 195? If so, do we need this condition? If not including 400 would make sense as well, maybe, wdyt? 🤔

Reply:
In line 210 the status code is set again. 

According to platform team 
>  If it's 4xx, it's a "[client error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses)" and this should generally be at the debug level.